src/Commands/ProjectRepairCommand.php

```diff
@@ -162,10 +162,8 @@ public function repair(
 
             ->addCode(
                 function ($state) use ($site_name) {
-                    $secretValues = [
-                        'token' => $this->git_provider->token($this->git_provider->tokenKey())
-                    ];
-                    $this->writeSecrets("{$site_name}.dev", $secretValues, false, 'tokens.json');
+                  $secretValues = $this->git_provider->getSecretValues();
+                  $this->writeSecrets("{$site_name}.dev", $secretValues, false, 'tokens.json');
```

Comment:
@aaronbauman won't `build:project:repair` miss the provider specific additions to `tokens.json`? `getSecretValues` only returns `token`.

Reply:
Ah, wait nevermind, it is calling `getSecretValues` for the provider, not the method from the base class

Reply:
`BitbucketProvider::getSecretValues()` returns `user` and `password` additionally, so should work.

https://github.com/pantheon-systems/terminus-build-tools-plugin/pull/268/files#diff-a39100c401681ff242854edb6766562aR147